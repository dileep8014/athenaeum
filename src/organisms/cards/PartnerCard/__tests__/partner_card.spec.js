import React from 'react';

// import sinon from 'sinon';
import renderer from 'react-test-renderer';

import PartnerCard from 'organisms/cards/PartnerCard';

describe('<PartnerCard />', () => {
  // let defaultProps;
  //
  // beforeEach(() => {
  //   defaultProps = {
  //     // Add default props here
  //   };
  // });

  it('renders correctly', () => {
    const actual = renderer.create(<PartnerCard />).toJSON();

    expect(actual).toMatchSnapshot();
  });
});
