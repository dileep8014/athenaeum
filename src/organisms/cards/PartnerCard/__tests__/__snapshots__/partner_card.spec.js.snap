// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PartnerCard /> renders correctly 1`] = `
<div
  className="partner-card"
>
  <div
    className="cell"
  >
    <span
      className="icon-wrapper"
      onClick={undefined}
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        alt="pencil"
        className="img"
        data-src={undefined}
        src="https://static.policygenius.com/svgs/pencil.svg"
        title="pencil"
      />
    </span>
  </div>
  <div
    className="cell"
  >
    <h2
      className="label"
    >
      BBB Rating
    </h2>
    <span
      className="a5"
    >
      A
    </span>
  </div>
  <div
    className="cell"
  >
    <button
      className="button info outline"
      disabled={undefined}
      onClick={undefined}
      type="button"
    >
      Shop Now
    </button>
  </div>
</div>
`;
